### ISSUE TITLE
Maximum Volume with MacBookAir M1 

### ISSUE THREAD

**Issue by PhilPhil2000** [ExistentialAudio/BlackHole#320]
Since yesterday I try to use BlackHole 2ch as the main output device on my MacBookAir M1.

The master level at 0.8 has an equal db loudness in the room as with the external headphone outputs (to PA) at 0.3,
therefore the maximum loudness of blackhole isn't really "rockin".
I also have to mention that in the chain before BlackHole I use AULab to bring some EQ in and in AULab I already maxed out input and ouput channel by + 6db.

Is there any chance to increase the output level?

**Comment by devinroth** [comment]
Is BlackHole volume all the way up?

Devin

**Comment by PhilPhil2000** [comment]
Hey Devin,

when BlackHole (set as output device) is all the way up it's okay but not loud enough to have "fun". ;)

In comparison the normal output devices of the macbook I never used at max.

Greetings
Phil

**Comment by devinroth** [comment]
I don’t really understand the issue.

BlackHole set at 1.0 0dB exactly passes the output to the input without changing any volume.

Are you asking for it to be higher than 100%? If so that will potential cause distortion and is something I will not do.

Devin

**Comment by PhilPhil2000** [comment]
I use BlackHole as system output !

**Comment by devinroth** [comment]
You want the volume to go higher than 100%?

Devin

**Comment by PhilPhil2000** [comment]
Yes...that could be a solution.
or to bring all system output to blackhole and from there to the a "normal" output device (I don't use it for recording my system output but for bringing in a EQ for room sound optimization)

**Comment by devinroth** [comment]
System Output -> Speakers = System Output -> BlackHole -> Speakers as long as BlackHole volume = 1.0 and Speakers volume remains the same. That why I don't understand why this is even an issue. 

If you want an additional boost add gain and a compressor in your signal path along with the EQ.

**Comment by PhilPhil2000** [comment]
"System Output -> Speakers = System Output -> BlackHole -> Speakers as long as BlackHole volume = 1.0 and Speakers volume remains the same." The issue is that the Speakers Volume does NOT remain the same but is decreased by a lot.

**Comment by devinroth** [comment]
BlackHole isn't adjusting your speaker volume. 

Just to prove my point here is a quick test that I did. With the volume of BlackHole set at 1.0 the output and input are identical. In every test the volume remains exactly the same.. 

<img width="389" alt="Screen Shot 2021-02-26 at 9 07 40 PM" src="https://user-images.githubusercontent.com/13010961/109376052-b39fcb00-7876-11eb-8b25-6cc633f29016.png">

<img width="1050" alt="Screen Shot 2021-02-26 at 9 04 26 PM" src="https://user-images.githubusercontent.com/13010961/109376017-7b988800-7876-11eb-891d-f129b42357ab.png">

I'm not saying that you're not having an issue, I'm just saying that it has absolutely nothing to do with BlackHole.

**Comment by PhilPhil2000** [comment]
Hey Devin,
thanks a lot for your effort - I really appreciate it.

After checking every possible knob and slider again I found the origin of my confusion/issue.

The volume +/- on my keyboard increases or decreases the master-level of BlackHole as it is set as output device.
And the master-level of external headphone audio device regulates the input for BlackHole
and has to be adjusted manually by mouse when blackhole is output device. 

I thought by setting BlackHole as output device it won't be affected by the master stream parameters of the other audio devices - there I went wrong.

So if I set the external headphone audio device level to 1.0 everything is fine
and hopefully I will always remember to decrease it before hitting play when not using blackhole. ;)

Best Regards,
Phil
